packages/wasm/crate/src/planner.rs

```diff
@@ -202,28 +206,12 @@ pub async fn plan_transaction(
         }
     };
 
-    // Decode the gas fee token into an `Id` type
-    let alt_gas: Id = Id::decode(gas_fee_token)?;
-
-    // Check if the decoded gas fee token is different from the staking token asset ID.
-    // If the gas fee token is different, use the alternative gas fee token with a 10x
-    // multiplier.
-    if alt_gas != *STAKING_TOKEN_ASSET_ID {
-        gas_prices = GasPrices {
-            asset_id: alt_gas,
-            block_space_price: gas_prices.block_space_price * 10,
-            compact_block_space_price: gas_prices.compact_block_space_price * 10,
-            verification_price: gas_prices.verification_price * 10,
-            execution_price: gas_prices.execution_price * 10,
```

Comment:
gas fee token price multiplier will instead be determined by governance proposals, rather than arbitrarily setting a 10x multiplier.